README.md

```diff
@@ -28,7 +28,30 @@ You should be able to get a good idea as to how other people are customising the
 
 ### Offline usage
 
-If you want to use an existing config available in `nf-core/configs`, and you're running on a system that has no internet connection, you'll need to download the config file and place it in a location that is visible to the file system on which you are running the pipeline. You can then run the pipeline with the `-c` parameter - see [Testing](#testing) for example.
+If you want to use an existing config available in `nf-core/configs`, and you're running on a system that has no internet connection, you'll need to download the config file and place it in a location that is visible to the file system on which you are running the pipeline. Then run the pipeline with `--custom_config_base`
+or `params.custom_config_base` set to the location of the directory containing the repository files:
+
+```bash
+## Download and unizp the config files
+cd /path/to/my/configs
+wget https://github.com/nf-core/configs/archive/master.zip
+unzip master.zip
+
+## Run the pipeline
+cd /path/to/my/data
+nextflow run /path/to/pipeline/ --custom_config_base /path/to/my/configs/configs-master/
+```
+
+Alternatively, instead of using the configuration profiles from this repository, you can run your
+pipeline directly calling the single institutional config file that you need with the `-c` parameter.
+
+```bash
+## Example command for nf-core/rnaseq
+nextflow run /path/to/pipeline/ -c /path/to/my/configs/configs-master/conf/my_config.config ..
+```
+
+> Note that the nf-core/tools helper package has a `download` subcommand download required
```

Comment:
I think you should probably emphasise the fact that this will download the Singularity containers, and the pipeline code too. Much much easier!